public/css/aboutus.css

```diff
@@ -0,0 +1,114 @@
+body {
+  margin: 0px;
+  padding: 0px;
+  height: 100vh;
+  width: 100%;
+  background-color: #f7f7f7;
+}
+/* start header */
+header {
+  height: 40px;
+  background-color: #752223;
+  display: flex;
+  justify-content: flex-end;
+}
+.headerForm {
+  display: flex;
+  align-items: center;
+  align: left;
+  margin-right: 10px;
+}
+
+input {
+  background-color: #742223;
+  border: 0;
+  color: white;
+  font-family: Roboto;
+  font-style: Regular;
+  font-size: 15px;
+}
+
+input:hover {
```

Comment:
it's better to use classes :D

Reply:
this issue for contact us not for about us